documentation/modules/ROOT/pages/transformations/timezone-converter.adoc

```diff
@@ -0,0 +1,181 @@
+            "xmin": null,
+            "id": 100
+        },
+        "op": "c",
+        "ts_ms": 1559033904863,
+        "event_timestamp": 1626102708861
+    }
+----
+====
+
+[[advanced-example-timezone-converter]]
+== Example: Advanced configuration
+
+For example, to convert the `created_at`, and `updated_at` timestamp fields in an event record from UTC to a fixed offset of `+05:30`, configure the `include.fields` configuration option as follows:
+
+[source]
+----
+transforms=convertTimezone
+transforms.convertTimezone.type=io.debezium.transforms.TimezoneConverter
+transforms.convertTimezone.converted.timezone=+05:30
+transforms.convertTimezone.include.fields=source:customers:created_at,customers:updated_at
+----
+
+In the same the way, you can exclude specific timestamp fields from timezone conversion. For example, to exclude the `updated_at` timestamp field in an event record from timezone conversion, configure the `exclude.fields` configuration option as follows:
+
+[source]
+----
+transforms=convertTimezone
+transforms.convertTimezone.type=io.debezium.transforms.TimezoneConverter
+transforms.convertTimezone.converted.timezone=+05:30
+transforms.convertTimezone.exclude.fields=source:customers:updated_at
+----
+
+[[timezone-converter-configuration-options]]
+== Configuration options
+
+The following table lists the configuration options that you can use with the `TimezoneConverter` SMT.
+
+.TimezoneConverter SMT configuration options
+[cols="14%a,40%a,10%a,10%a"]
+|===
+|Property
+|Description
+|Type
+|Importance
+
+|[[timezone-converter-converted-timezone]]<<timezone-converter-converted-timezone, `converted.timezone`>>
+|A string that specifies the target timezone to which the timestamp fields should be converted. The target timezone can be specified as a geographic timezone e.g. `America/New_York` or as a UTC offset e.g. `+02:00`.
+|string
+|high
+|[[timezone-converter-include-fields]]<<timezone-converter-include-fields, `include.fields`>>
+|A comma-separated list of rules that specify which fields should be included for timezone conversion, using one of the following formats:
+
+    `source:<tablename>` :: Matches Debezium change events with a source information block having the specified table name. All time-based fields in the matched table will be converted.
+    `source:<tablename>:<fieldname>` :: Matches Debezium change events with a source information block having the specified table name. Only the specified field name will be converted.
+    `topic:<topicname>` :: Matches events from the specified topic name, converting all time-based fields.
+    `topic:<topicname>:<fieldname>` :: Matches events from the specified topic name, converting only the specified field name.
+    `<matchname>:<fieldname>` :: Applies a heuristic matching algorithm to match against the source information block table name if present; otherwise, matches against the topic name. Conversion is applied only to the specified field name.
```

Comment:
```suggestion
    `<matchname>:<fieldname>` :: Applies a heuristic matching algorithm to match against the table name of the source information block, if present; otherwise, matches against the topic name. 
The SMT converts values for the specified field name only.
```